docs/security.md

```diff
@@ -186,7 +186,54 @@ configure those ports.
   </tr>
 </table>
 
+### HTTP Security Headers
+
+Apache Spark can be configured to include HTTP Headers which aids in preventing Cross 
+Site Scripting (XSS), Cross-Frame Scripting (XFS), MIME-Sniffing and also enforces HTTP 
+Strict Transport Security.
+
+<table class="table">
+    <tr><th>Property Name</th><th>Default</th><th>Meaning</th></tr>
```

Comment:
I think in Spark we follow 2 space indent for html code. You could refer to other docs.

Reply:
@jerryshao Fixed the indentation issue as per other Spark docs. Thanks for pointing that out.